dependency.lic

```diff
@@ -205,8 +205,8 @@ class ScriptManager
     @paste_bin_token = 'dca351a27a8af501a8d3123e29af7981'
     @paste_bin_url = 'https://pastebin.com/api/api_post.php'
     @firebase_url = 'https://dr-scripts.firebaseio.com/'
-    @status_repo = Settings['status_repo'] || 'rpherbig/dr-scripts'
-    @status_branch = Settings['status_branch'] || 'master'
+    @status_repo = 'rpherbig/dr-scripts'
```

Comment:
If anyone gets stuck on `1.7.4`, you might have them set `Settings['status_branch']` and `Settings['status_repo']` before trying to download a new `dependency.lic`

Reply:
That's a really great idea. Thank you. I always appreciate your thoughtful comments when you review code.